cas-management-webapp/src/main/webapp/WEB-INF/spring-configuration/log4jConfiguration.xml

```diff
@@ -34,12 +35,9 @@
         Deployers should not have to edit this file.
     </description>
 
-    <bean id="log4jInitialization" class="org.springframework.beans.factory.config.MethodInvokingFactoryBean"
-        p:targetClass="org.springframework.util.Log4jConfigurer" p:targetMethod="initLogging" p:arguments-ref="arguments"/>
-    
-    <util:list id="arguments">
-       <value>${log4j.config.location:classpath:log4j.xml}</value>
-       <value>${log4j.refresh.interval:60000}</value>
-    </util:list>
+    <bean id="log4jInitialization" class="org.jasig.cas.util.CasLoggerContextInitializer"
```

Comment:
Directly inside the log4j2.xml file: https://github.com/Jasig/cas/pull/870/files#diff-1247a897c9e6da7eb28c8ca9bd007b73R23

with `<Configuration monitorInterval="60">`. the interval is in seconds.